pkg/controller/direct/cloudquota/quotaadjustersettings_controller.go

```diff
@@ -124,8 +124,8 @@ func (a *apiQuotaAdjusterSettingsAdapter) Find(ctx context.Context) (bool, error
 // Create is not supported for QuotaAdjusterSettings. This resource is managed by GCP implicitly.
```

Comment:
We might want to say something like "this function should not be reachable, because Find always returns an object"